client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -891,10 +891,18 @@ Params withFields(String[] fields) {
             return this;
         }
 
+        /**
+         * @deprecated As of 2.0, because supporting inclusive language, replaced by {@link #withClusterManagerTimeout(TimeValue)}
+         */
+        @Deprecated
         Params withMasterTimeout(TimeValue masterTimeout) {
             return putParam("master_timeout", masterTimeout);
         }
 
+        Params withClusterManagerTimeout(TimeValue clusterManagerTimeout) {
+            return putParam("cluster_manager_timeout", clusterManagerTimeout);
+        }
```

Comment:
Note: this is the core change for this PR, using parameter "cluster_manager_timeout" instead of "master_timeout" to build REST requests can avoid the deprecation warning.